api/migrations/0010_auto_20210402_2245.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.4 on 2021-04-03 02:45
+
+import json
+
+from django.db import migrations
+
+
+def add_interests(apps, schema_editor):
+    # We can't import the Interest model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Interest = apps.get_model("api", "Interest")
+
+    # Add all predefined interests into model.
+    with open("api/data/interests.json") as fileData:
+        data = json.load(fileData)
+        for interestData in data:
+            try:
+                category = interestData["category"]
+                title = interestData["title"]
+                interest = Interest(category=category, title=title)
+                interest.save()
+            except KeyError:
+                # Interest does not follow category/title structure.
+                pass
```

Comment:
Log this error so we know when it happens. This will tell us if there is a mistake in the interests file. Silently ignoring errors is almost never a good idea. 
```suggestion
            except KeyError as e:
                print(f"Invalid key in interests.json: {e}")
```